setup.cfg

```diff
@@ -8,4 +8,4 @@
 [flake8]
 ignore = E501,E127,E128,E265,E301,E302,F403,E731
 max-line-length = 100
-exclude = .ropeproject,tests/*,src/*,env,venv
+exclude = .ropeproject,tests/*,src/*,env,venv,./node_modules/*
```

Comment:
The PR updating the [flake8] has already been merged [here](https://github.com/CenterForOpenScience/modular-file-renderer/pull/329/files). Please remove this change or fix conflicts.